<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Egge's IP Inquirer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>360</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>9</y>
         <width>681</width>
         <height>261</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>501</width>
         <height>121</height>
        </rect>
       </property>
       <property name="title">
        <string>Inquiries</string>
       </property>
       <widget class="QCheckBox" name="checkBoxIPv4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>IP address (ipv4)</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxEnv">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>211</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Environment variables</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxLocal">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Local IP addresses</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonInquire">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>310</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="tabletTracking">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Inquire</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
